bgpd/bgpd.h

```diff
@@ -305,9 +310,9 @@ struct graceful_restart_info {
 	/* Best route select */
 	struct event *t_route_select;
 	/* AFI, SAFI enabled */
-	bool af_enabled[AFI_MAX][SAFI_MAX];
+	bool af_enabled;
```

Comment:
Since this is moved `bgp->gr_info` per AFI/SAFI, then the naming could be adjusted also to just `enabled`?

Reply:
Although bgp->gr_info has been moved to per AFI/SAFI, af_enabled make sense since this tell us if GR is enabled for that AFi-SAFI IMO.